core/market/src/rest_api/provider.rs

```diff
@@ -21,6 +21,7 @@ pub fn register_endpoints(scope: Scope) -> Scope {
         .service(counter_proposal)
         .service(get_proposal)
         .service(reject_proposal)
+        .service(reject_proposal)
```

Comment:
Important: Duplicated?

Reply:
nice catch again. I was trying to remove duplicates ;) But will try harder!